src/tests/bignum_test.nr

```diff
@@ -496,15 +449,12 @@ fn test_add_modulus_limit() {
 fn test_add_modulus_overflow() {
     //(2^{254} - 1) + (p - 1) = 2^{254} + p
     // after subtracting modulus, result is 2^{254} will does not satisfy the range check applied when constructing a BigNum
-    let p: U60Repr<3, 2> = U60Repr::from(BN254_Fq_Params::get_params().modulus);
-    let two_pow_254_minus_1: U60Repr<3, 2> = U60Repr::from([
-        0xffffffffffffffffffffffffffffff,
-        0xffffffffffffffffffffffffffffff,
-        0x3fff,
-    ]);
-    let one = U60Repr::from([1, 0, 0]);
-    let a: Fq = BigNum { limbs: U60Repr::into(p + one) };
-    let b: Fq = BigNum { limbs: U60Repr::into(two_pow_254_minus_1) };
+    let p: [u128; 3] = BN254_Fq_Params::get_params().modulus;
+    let two_pow_254_minus_1: [u128; 3] =
+        [0xffffffffffffffffffffffffffffff, 0xffffffffffffffffffffffffffffff, 0x3fff];
+    let one = [1, 0, 0];
+    let a: Fq = BigNum { limbs: unsafe { __helper_add(p, one) } };
```

Comment:
This PR adds a lot of `unsafe` blocks which are going to throw warnings. Can you replace these with calls to a safe `add` function?